$OpenBSD: patch-src_main_c,v 1.12 2014/07/11 15:49:58 dcoppa Exp $

commit 5c30c6ca80e3694bbde5ba46c5a297f104d818dc
Author: Michael Stapelberg <michael@stapelberg.de>
Date:   Tue Jun 24 09:56:31 2014 +0200

add log messages for the detected DPI

Currently, the DPI is only logged when pango fonts are used, which is
often not the case when people report problems with the DPI detection.

--- src/main.c.orig	Sun Jun 15 19:12:43 2014
+++ src/main.c	Fri Jul 11 16:37:49 2014
@@ -558,6 +558,10 @@ int main(int argc, char *argv[]) {
     colormap = root_screen->default_colormap;
 
     DLOG("root_depth = %d, visual_id = 0x%08x.\n", root_depth, visual_id);
+    DLOG("root_screen->height_in_pixels = %d, root_screen->height_in_millimeters = %d, dpi = %d\n",
+         root_screen->height_in_pixels, root_screen->height_in_millimeters,
+         (int)((double)root_screen->height_in_pixels * 25.4 / (double)root_screen->height_in_millimeters));
+    DLOG("One logical pixel corresponds to %d physical pixels on this display.\n", logical_px(1));
 
     xcb_get_geometry_cookie_t gcookie = xcb_get_geometry(conn, root);
     xcb_query_pointer_cookie_t pointercookie = xcb_query_pointer(conn, root);
